frontend/archive/src/main/java/org/pytorch/serve/archive/model/ModelArchive.java

```diff
@@ -70,6 +70,11 @@ public static ModelArchive downloadModel(
 
         if (new File(url).isDirectory()) {
             return load(url, new File(url), false);
+        } else if (modelLocation.exists()) {
```

Comment:
@lxning how would we handle the case if there are multiple mar files in a directory? are we relying on user to have only one mar file in the /xxx/model_store/modelXXX directory? In this case we need to make sure we have documented it clearly. 

Do you we have any documentation on it?

Reply:
I understood this PR as no longer needing a `model_store` and just letting users link to various model files directy

Reply:
In model_store dir, existing code ignores the mar files in the subdir (ie.  /xxx/model_store/modelXXX).